packages/next/lib/find-pages-dir.ts

```diff
@@ -29,30 +29,27 @@ export function findPagesDir(
   appDir: string | undefined
 } {
   const pagesDir = findDir(dir, 'pages') || undefined
-  let appDir: undefined | string
+  const appDir = findDir(dir, 'app') || undefined
```

Comment:
Let's also have the warning for the case where `pages` is present and `app` is present but `experimental.appDir` is not enabled. 

Can be in a follow-up though.